### PR TITLE
add landing page message #1

### PR DESCRIPTION
**Pull request by cturner2001**

### CODE REVIEW

**Comment by renouchea** [comment]
Not sure about this one !!!

File: index.html [hunk 1]
```diff
@@ -1 +1 @@
-
+<h1>
```
Comment (renouchea) [inline index.html]: Not sure about<h1>!!!

Comment (cturner2001) [reply index.html]: it is correct

**Review by renouchea — CHANGES_REQUESTED** [review]
change to another header

**Review by renouchea — APPROVED** [review]

**Pull request merged**